cloud/kubernetes/helm/yugabyte/templates/service.yaml

```diff
@@ -31,6 +31,18 @@ apiVersion: v1
 kind: Service
 metadata:
   name: "{{ $endpoint.name }}"
+{{- if $root.Values.annotations }}
+  annotations:
+  {{ if eq .name "yb-master-service" }}
+    {{- if $root.Values.annotations.master }}
```

Comment:
can we add a check to make sure we have got a loadbalancer annotation, if not we don't need to add any annotation.